_overviews/tutorials/scala-on-android.md

```diff
@@ -0,0 +1,137 @@
+
+### How to run the app
+
+Building an  Android native image takes time, so we want to avoid doing it too often. Even before running the app for the first time, you should invest some  time in unit, component, and integration tests, so that if you change  something in your app you could still be sure it works correctly even  before any manual testing. Then, to check how your GUI looks like and  works, use:
+
+```
+mvn gluonfx:run
+```
+
+If everything looks fine, build the native image… but first, for your desktop:
+
+```
+mvn gluonfx:build gluonfx:nativerun
+```
+
+After all, we work on a cross-platform solution here. Unless you want to test features of your app that will only work on a  mobile device, you can first run it as a standalone desktop application. This will again let you test some layers of the app without actually  running it on an Android device. And then, if all looks good, or if you  decide to skip this step:
+
+```
+mvn -Pandroid gluonfx:build gluonfx:package
+```
+
+Successful execution of this command will create an APK file in the` target/client/aarch64-android/gvm` directory. Connect your Android phone to the computer with an USB  cable, give the computer permission to send files to the phone, and type `adb devices` to check if your phone is recognized. It should display something like this in the console:
+
+```
+> adb devices
+List of devices attached
+16b3a5c8	device
+```
+
+Now you should be able to install the app on the connected device with `adb install <path to APK>` and a moment later you should see a new icon on your device’s main  screen. When you click on the icon, it should open approximately the  same screen as the desktop version of your app.
+
+Installation might not work for a  number of reasons, one of the most popular being that your Android  simply does not allow installing apps this way. Go to Settings, find  „Developers options”, and there enable „USB debugging” and „Install via  USB”. 
+
+If everything works and you see the app’s screen on your device, type `adb logcat | grep GraalCompiled` to see the log output. Now you can click the button with the magnifying glass icon on the app’s  screen and you should see `"log something from Scala"`  printed to the console. Of course, before you write a more complex app, please  look into plugins in the IDE of your choice that can display logs from  `adb logcat` in a better way.
```

Comment:
Could you maybe add a couple of links to such plugins (maybe for IntelliJ and VS Code)?

Reply:
done